globalsSelectors.ipynb

```diff
@@ -117,13 +132,29 @@
     "                checkboxesPlayersSelected.multi_select,\n",
     "                checkboxesAlternativeSelected.multi_select\n",
     "            ])\n",
+    "        ]),\n",
+    "        VBox([\n",
+    "            widgets.HTML(\"Use the Selectors widgets to select the players and items you want to analyse\"),\n",
+    "            HBox([\n",
+    "                checkboxesPlayersSelected.multi_select,\n",
+    "                checkboxesItemInteractedSelected.multi_select\n",
+    "            ])\n",
+    "        ]),\n",
+    "        VBox([\n",
+    "            widgets.HTML(\"Use the Selectors widgets to select the players and accessed zone you want to analyse\"),\n",
+    "            HBox([\n",
+    "                checkboxesPlayersSelected.multi_select,\n",
+    "                checkboxesAccessibleSelected.multi_select\n",
+    "            ])\n",
     "        ])\n",
     "    ])\n",
-    "    tabcheckboxes.set_title(0,\"Games started and Completed\")\n",
-    "    tabcheckboxes.set_title(1,\"Videos seen and skipped\")\n",
+    "    tabcheckboxes.set_title(0,\"Start/Complete games\")\n",
+    "    tabcheckboxes.set_title(1,\"Videos\")\n",
     "    tabcheckboxes.set_title(2,\"Progress\")\n",
     "    tabcheckboxes.set_title(3,\"Completable\")\n",
     "    tabcheckboxes.set_title(4,\"Alternative\")\n",
+    "    tabcheckboxes.set_title(5,\"Interactions\")\n",
+    "    tabcheckboxes.set_title(6,\"Accessibles\")\n",
```

Comment:
With the current tab duplication, this tab only allows to select players/accessibles, but no visualization is actually shown below (see comment in xAPISGProcessor notebook).